collector/topological_inventory.py

```diff
@@ -29,6 +30,10 @@
 )
 
 
+DATA_COLLECTION_TIME = prometheus_metrics.METRICS['data_collection_time']
+ALERTIBLE_DATE_SIZE = 100 * 10**6
```

Comment:
can you use a constant name that is more obvious of what it means?

Reply:
The constant has been removed, since in a way, it was related to labeling.

We are not labeling anymore, so no need for this constant